### ISSUE TITLE
font-size rules set null

### ISSUE THREAD

**Issue by fuzegit** [scssphp/scssphp#84]
Hi
Thanks for the work!

PHP 5.6.40, 7.3.15, 7.4.3

I compile bootstrap
https://github.com/twbs/bootstrap/blob/master/scss/bootstrap.scss

`./pscss /path-to/scss/bootstrap.scss > bs.css`

There is such a font-size problem:

![Выделение_008](https://user-images.githubusercontent.com/13233234/75187395-05d97800-575b-11ea-8ad9-33b5f9c289f3.png)

**Comment by Cerdic** [comment]
I can reproduce, but only with the Bootstrap master branch, not with the last release 4.4.1

**Comment by fuzegit** [comment]
With the last release 4.4.1 there really is no such problem.
I installed through
`git clone https://github.com/twbs/bootstrap.git`

**Comment by Cerdic** [comment]
this is linked to commit https://github.com/twbs/bootstrap/commit/cd077cd59

**Comment by fuzegit** [comment]
Everything is fine, but now another problem)

![Выделение_009](https://user-images.githubusercontent.com/13233234/75241598-6f976780-57d7-11ea-9fff-c464249f8f3d.png)

`$breadcrumb-font-size:              null !default;`

**Comment by Cerdic** [comment]
what do you mean by "everything is fine"? 
* If you are now working with v4.4.1 and having another problem: please open a new issue, this is probably not linked;
* If you are still working on master branch, nothing is fine as we still have to solve the issue

**Comment by fuzegit** [comment]
Yes, the problem is resolved

**Comment by Cerdic** [comment]
Problem is not solved: it's still occuring with the master version of BS and we need to figure out if it's an issue from scssphp or BS, so i will keep this open

**Comment by Cerdic** [comment]
Minimal test case to reproduce:
```
// Helper function to get the formatted non-responsive value
@function rfs-value($values) {
  // Convert to list
  $values: if(type-of($values) != list, ($values,), $values);

  $val: '';

  // Loop over each value and calculate value
  @each $value in $values {
    $val: $val + ' ' + $value;
  }

  // Remove first space
  @return unquote(str-slice($val, 2));
}

// RFS mixin
@mixin rfs($values, $property: font-size) {
  @if $values != null {
    $val: rfs-value($values);
    //$fluidVal: rfs-fluid-value($values);
		#{$property}: $val;
  }
}

.test {
	@include rfs(1rem);
}
```

Output with scssphp:
```
.test {
  font-size: 1rem null; 
}
```

Ouput with sassmeister:
```
.test {
  font-size: 1rem;
}
```

**Comment by Cerdic** [comment]
this is fixed by 4ebc5e14
